app/controllers/users_controller.rb

```diff
@@ -29,6 +29,17 @@ def create
     end
   end
 
+  def update
+    @user = authorize User.find_by(id: params[:id])
+    if @user.update(user_params)
```

Comment:
Ugye eddig a user_params csak a create-ben volt hívva. Biztos, hogy az update-ben is hagyatkozhatunk rá a jelenlegi formájában? Nem léphet fel pl egy olyan szituáció, hogy a saját user-emet edit-elve be tudok küldeni egy `{ admin: true }` -t?

Ha ez egy security risk, akkor csináljunk ide egy attribute policy-t, ami a user_params-ban kifilterezi a veszélyes attribútumokat.

Reply:
ezt már megírtuk, csak admin adhat adminnak admin jogot, saját magának nem fog tudni

Reply:
Igen, de szerintem nincs itt meghívva. A user_params-ban én ilyesmit várnék, hogy  
1. require  
2. permit  
3. filter <= ez hiányzik  
Most hirtelen azt látom, hogy eddig mintha csak a standalone `valid_admin?` helpert használtátok volna. Írok mintakódot, hogy mit lehetne itt:
```ruby
  def user_params
    eddig_ez_volt = params
      .require(:user)
      .permit(:dangerous, :not_dangerous)
    policy([:attributes, :user]).filter(eddig_ez_volt)
  end
```